src/db.c

```diff
@@ -1944,6 +1944,72 @@ static struct db_sys_list *_db_rule_gen_64(const struct arch_def *arch,
 	return NULL;
 }
 
+static int _db_rule_gen_arg_32(struct db_sys_list *s_new,
```

Comment:
This function also needs a comment block at the top like all the other functions in libseccomp.